tests/test_setup_kubernetes_job.py

```diff
@@ -358,6 +384,7 @@ def simple_create_application_object(
                 service="kurupt",
                 instance="garage",
                 git_sha="1",
+                image_version=None,
```

Comment:
would be nice to see a test case where image_version isn't None.

Reply:
I've modified `test_setup_kube_deployment_create_update` so that the test cases set `image_version` by default, which also lets us test that an update happens when `image_version` gets changed, let me know if that works 👍